src/lib/youTubeApi.js

```diff
@@ -0,0 +1,17 @@
+const key = 'AIzaSyCbnwRIBvWCoQGaLIOFUx4rhLHg-Sb1a8s';
+
+export async function fetchVideos(keyWord) {
+  const apiBaseUrl = `https://content-youtube.googleapis.com/youtube/v3/search?part=id&part=snippet&maxResults=25&q=${keyWord}&key=${key}`;
+
+  return fetch(apiBaseUrl)
```

Comment:
What if this fn is a custom hook instead ? Do you see a better/cleaner usage for this functionality at the other components using it?